### ISSUE TITLE
The status of links in article/note should be recognized as visited and unvisited

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3312]
**Environment:** 7.x

**Step of reproduction:**

- Add an article/note then add a link and a mention in its content
- Visit the link and the mentioned user

**Current behavior:**

The is no difference of color between the visited and unvisited status of link and mention

**Expected behavior:**

There must be a difference between visited and unvisited status:

- Unvisited link color: #006EDB
- Visited link color: #9B589B
